sdk/lib/crt/math/exp2.c

```diff
@@ -8,6 +8,6 @@ exp2(
     _In_ double x)
 {
     /* This below avoids clang to optimize our pow call to exp2 */
-    static const TWO = 2.0;
+    static const double TWO = 2.0;
```

Comment:
Does this still avoid clang to optimize the call?
Why did you change this?

Reply:
To be honest, I don't know how the old version even compiles. It's not like types are optional in C.
(but yes, please verify that the generated assembly code is the same on all compilers)

Reply:
Spoiler alert: this change does change the generated code.
To be fair, the new code looks better... but the gist is, this needs a test.

Reply:
missing type?

Reply:
Oh, my bad :-( I don't even know how I could have missed that.

Reply:
> Oh, my bad :-( I don't even know how I could have missed that.

-wImplicit is enabled by -Wall in both GCC & clang builds. But only clang actually uses this file, and -Werror is disabled here, that's how things slip through.